### ISSUE TITLE
search engine keywords do not work

### ISSUE THREAD

**Issue by glacambre** [atlas-engineer/next#319]
I followed the procedure in https://github.com/atlas-engineer/next/blob/master/documents/MANUAL.org#searching-via-search-engine but `C-l` `wiki parrot` `Return` does not open wikipedia. Instead, the default search engine is used with `wiki parrot` as search query.

**Comment by alexanderbarbosa** [comment]
Can you share the search part of your next config.

**Comment by jmercouris** [comment]
Can you please share the results of: `(get-default 'window 'search-engines)`?

**Comment by alexanderbarbosa** [comment]
Sorry to budge in, but here goes:

```lisp
=> (("default" . "https://duckduckgo.com/?q=~a")
    ("qd" . "http://quickdocs.org/search?q=~a")
    ("yt" . "https://www.youtube.com/results?search_query=~a")
    ("wiki" . "https://en.wikipedia.org/w/index.php?search=~a"))
```

**Comment by glacambre** [comment]
I get the default list of search engines: https://github.com/atlas-engineer/next/blob/41b6960004d7c9af08806064d7d86ba8f9c5568a/source/remote.lisp#L27-L28

Changing this list as described in the [manual](https://github.com/atlas-engineer/next/blob/41b6960004d7c9af08806064d7d86ba8f9c5568a/source/remote.lisp#L27-L28) works for the default search engine but doesn't change anything for the others.

**Comment by Ambrevar** [comment]
Thanks for reporting.
There was an issue indeed with the raw input candidate: the non-breaking
space was not replaced by a space, which led `parse-url` to see
`wiki foo` as one word.

Fixed in d122dea11d537916fdd500e13f7ab8ad7a6a0c2b.

**Comment by glacambre** [comment]
Seems to be working correctly with d122dea, thank you!
